src/ToggleButton/__tests__/__snapshots__/ToggleButton.test.js.snap

```diff
@@ -13,8 +13,8 @@ exports[`<ToggleButton /> should call componentDidUpdate and setState 1`] = `
   width: 3.8rem;
   height: 2rem;
   border-radius: 1rem;
-  background: hsl(200,74%,46%) linear-gradient( 0deg, hsla(0,100%,100%,0) 0%, hsla(0,100%,100%,0.1) 100% );
-  border: 1px solid hsl(200,74%,46%);
+  background: hsl(207,22%,90%);
```

Comment:
Why does the linear gradient missing ?

Reply:
@spartDev same here, the background hsl value is not the same between the old snapshot and the new